### PR TITLE
Check errors in ENSName input value

### PR DESCRIPTION
**Pull request by peripheralist**
## What does this PR do and why?

- Updates ENSName input validator to prevent disallowed inputs
  - No spaces
  - No uppercase letters
- Updates the ConfigureHandle modal that ensures normalization of ENS name string value, and disables submit button when input value is invalid

## Screenshots or screen recordings

https://user-images.githubusercontent.com/79433522/177679995-6decc1d8-dbb1-4a92-a1ab-afc71454f9dd.mov

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by aeolianeth** [comment]
@johnnyd-eth can you please review this for Peri?

**Comment by johnnyd-eth** [comment]
Apologies for letting this one slip through the cracks. Will get on it first thing tomorrow

File: src/components/v2/V2Project/V2ReconfigureProjectHandleDrawer.tsx [hunk 1]
```diff
@@ -157,8 +160,23 @@ export function V2ReconfigureProjectHandleDrawer({
...
-          <Button htmlType="submit" loading={loadingSetENSName} type="primary">
+          <FormItems.ENSName
+            name="ensName"
+            hideLabel
+            formItemProps={{ rules: [{ required: true }] }}
+            onChange={() => {
+              ensNameForm
+                .validateFields()
+                .then(() => setEnsNameIsValid(true))
+                .catch(() => setEnsNameIsValid(false))
```
Comment (johnnyd-eth) [inline src/components/v2/V2Project/V2ReconfigureProjectHandleDrawer.tsx:171]: Love this

**Review by johnnyd-eth — APPROVED** [review]

**Comment by johnnyd-eth** [comment]
Great validation additions Peri, LGTM

**Pull request merged**